src/context/StoreContext.js

```diff
@@ -87,6 +87,20 @@ export default function StoreContextProvider(props) {
     localSetBibleReference(obj)
   }
 
+  const setNewBibleBook = (bId) => {
+    const newBibleBookSourceId = "new-Biblebook"
+    const _bibleRef = bibleReference
```

Comment:
This is only creating a soft copy of `bibleReference` in which case the `_bibleRef.bookId = ...` code below will modify both `_bibleRef` and `bibleReference`. Is that what you intended to do? If so could we remove the `_bibleRef` variable?